hack/util.sh

```diff
@@ -46,10 +46,10 @@ function cleanup_openshift() {
 	# really have it smack people in their logs.  This is a severe correctness problem
 	grep -a5 "CACHE.*ALTERED" ${LOG_DIR}/openshift.log
 
-	os::cleanup::dump_etcd
 	os::cleanup::dump_container_logs
 
 	if [[ -z "${SKIP_TEARDOWN-}" ]]; then
+		os::cleanup::dump_etcd
```

Comment:
This seems unrelated to the other changes, but looks fine as long as it doesn't break anyone's workflow